frontend/src/components/RequestRideModal/RequestRideInfo.tsx

```diff
@@ -366,10 +363,13 @@ const RequestRideInfo = ({
             aria-labelledby="dropoffLabel dropoffTime"
             ref={register({
               required: true,
-              validate: (dropoffTime: any) => {
-                const dropoffTi = dropoffTime;
+              validate: (dropoffTime: string) => {
                 const pickupTi = getValues('pickupTime');
```

Comment:
I think you can consider changing the variable name here so it's more clear what this variable is being used for.